### PR TITLE
config: change escape single quotes policy for crowdin

### PR DESCRIPTION
**Pull request by strkkk**
change escape policy to disable auto escape

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ files:
   - source: /**/messages.properties
     translation: /**/messages_%two_letters_code%.properties
     update_option: update_without_changes
-    # Escape single quote with another single quote
-    escape_quotes: 1
+    # Do not escape single quote
+    escape_quotes: 0
```
[the remark this answers is not in the document]
Comment (strkkk) [reply crowdin.yml:6]: @romani asked in this comment. 
https://github.com/checkstyle/checkstyle/issues/6191#issuecomment-967744149
About bug - I didn't submit anything, but I can.

[the remark this answers is not in the document]
Comment (romani) [reply crowdin.yml:6]: I am confused, do we need to escape single quote or not ?

[the remark this answers is not in the document]
Comment (strkkk) [reply crowdin.yml:6]: I think it is better to avoid autoescape and escape it like we do now

[the remark this answers is not in the document]
Comment (romani) [reply crowdin.yml:6]: @strkkk , can you us a favor:
Update this PR description with details why it is ok to not escape. Examples might be best way.
Update commit of this PR to reference PR, linkage to description.

[the remark this answers is not in the document]
Comment (rnveach) [reply crowdin.yml:6]: @romani for your first question,
https://docs.oracle.com/javase/6/docs/api/java/text/MessageFormat.html
>Within a String, '' (two single quotes ) represents a single quote. A QuotedString can contain arbitrary characters except single quotes; the surrounding single quotes are removed. An UnquotedString can contain arbitrary characters except single quotes and left curly brackets. Thus, a string that should result in the formatted message '{0}' can be written as '''{'0}'' or '''{0}'''.

[the remark this answers is not in the document]
Comment (romani) [reply crowdin.yml:6]: based on experiment at https://github.com/checkstyle/checkstyle/issues/6191#issuecomment-939486485

if we whant to print `'` we need to do doubling - `''`.
single `'` helps us to hack `{@inheritDoc}` rendering, as single `'` is ignored.

so here is mix:
if in language `'` used for language needs, it is better to double it, it is REQUIRED.
we use `'` for hacks to escape `{....}`. We might find another workaround.

Resolution: we need doubling of single quote.

@strkkk , @rnveach , please confirm my ideas is good.

[the remark this answers is not in the document]
Comment (romani) [reply crowdin.yml:6]: but this tool do doubling all the time:
image
that is not good.

https://github.com/checkstyle/checkstyle/pull/10974/files#diff-49a68b76c344f3302f0fe38bb4c8cd1cfc6003817f2f731baa5de2c11515507dL12

[the remark this answers is not in the document]
Comment (romani) [reply crowdin.yml:6]: So based in that thing tool does by doubling is not good at all. We can not use this option. 
So it should be disabled.

**Review by romani — APPROVED** [review]

File: crowdin.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (strkkk) [reply crowdin.yml:6]: for some cases it adds escape, for some cases it doesnt. it is better to manage escapes by ourselves

**Review by rnveach — APPROVED** [review]

**Pull request merged**